### PR TITLE
added kube definition

### PR DESCRIPTION
**Pull request by scouball**
Added a kube file for simpler deployment and the needed documentation.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -81,6 +80,17 @@ corona-sign-in flask db upgrade`
...
+
+1. Build the image
+    `podman build -t corona-sign-in .`
+2. Load the kube file
+    `podman play kube kube.yml`
+3. Migrate the database
+    `podman container run --rm --pod corona-sign-in -e CORONA_SIGN_IN_DATABASE_URI="postgresql://corona-sign-in:pass@localhost/corona-sign-in" -e CORONA_SIGN_IN_SECRET_KEY="irrelevant" corona-sign-in flask db upgrade`
+4. Restart the application container
+    `podman podman restart corona-sign-in-app`
+
```
Comment (voidus) [inline Readme.md:93]: Do we really have to restart the container? I wouldn't know why.

File: kube.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    name: POSTGRES_DB
+    value: corona-sign-in
+
+
+spec:
+  containers:
+    # necessary when using caddy as a reverse proxy for automatic route creation 
+    # - name: corona-sign-in-route
+    #   image: localhost/podman_caddy
+    #   command: [ 'add', '--forward', 'corona.gaengeviertel.tk:corona-sign-in:8080', '--update', '5' ]
```
Comment (voidus) [inline kube.yml:23]: If this is needed for the gaengeviertel deployment, I'd actually put it in here, spare you the work of changing it for the deployments.
Maybe instead add a comment that people can open an issue if they need it generalized. Your call.

File: kube.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    - name: corona-sign-in-db
+      image: postgres
```
Comment (voidus) [inline kube.yml:26]: We should probably pin the postgres version here

File: Readme.md (hunk 1 above)
Comment (voidus) [reply Readme.md:93]: Tested and it seems to work locally without the restart

File: kube.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: corona-sign-in
+
+x-env:
```
Comment (voidus) [inline kube.yml:6]: I just learned about about yaml anchors and it inspires the worst kind of awe

**Comment by voidus** [comment]
Added a few minor comments, but lgtm. Feel free to merge when you feel it's ready.

File: kube.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        - name: CORONA_SIGN_IN_SECRET_KEY
+          value: irrelevant
+      # only necessary for development
+      ports: 
+        - containerPort: 8080
+          hostPort: 8080
+          protocol: tcp
+      volumeMounts:
+        - name: data
+          mountPath: /app
```
Comment (voidus) [inline kube.yml:51]: Why are we mounting a folder in the app? I think it's self-contained.

Comment (voidus) [reply kube.yml:51]: Okay, I'm really bad at reading today, disregard this.

**Comment by voidus** [comment]
Actually, I have to change my position. We need to discuss whether this is a development setup or a production setup. I assumed this was for production, right? If so, we should remove the dev parts.

**Comment by scouball** [comment]
Yeah, think so as well. Didn't put that much thought into this at that time. For simplicity I would put a development file into this repo and keep the prod version in our private github. Do you approve?

**Comment by voidus** [comment]
Of course, thank you :) and please excuse my late-night ramblings...

Feel free to merge this whenever you feel it's ready.

**Pull request merged**